contracts/script/GenerateAlloc.s.sol

```diff
@@ -44,7 +44,8 @@ contract GenerateAlloc is Script {
 
     string internal dumpPath = getDumpPath();
     bool public saveState = true;
-    uint256 public constant MAINNET_CHAIN_ID = 0; // TBD
+    uint256 public constant MAINNET_CHAIN_ID = 1514; // TBD
+    bool private constant ALLOCATE_10K_TEST_ACCOUNTS = false;
```

Comment:
Add a comment so users know to change the value to perform the test.

Reply:
Added